spec/numberformat.html

```diff
@@ -22,27 +22,35 @@ <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefa
         1. Let _mnsd_ be ? Get(_options_, *"minimumSignificantDigits"*).
         1. Let _mxsd_ be ? Get(_options_, *"maximumSignificantDigits"*).
         1. Set _intlObj_.[[MinimumIntegerDigits]] to _mnid_.
-        1. If _mnsd_ is not *undefined* or _mxsd_ is not *undefined*, then
-          1. Set _intlObj_.[[RoundingType]] to ~significantDigits~.
+        1. If _mnsd_ is not *undefined* or _mxsd_ is not *undefined*, set _hasSd_ to *true*; else, set _hasSd_ to *false*.
+        1. If _mnfd_ is not *undefined* or _mxfd_ is not *undefined*, set _hasFd_ to *true*; else, set _hasFd_ to *false*.
```

Comment:
Any reason why both of these conditions are on a single line instead of being split across two?

Reply:
Because I find it more readable to write

1. If _condition_, set _a_ to *true*; else, set _a_ to *false*.
1. If _condition_, set _b_ to *true*; else, set _b_ to *false*.
1. If _condition_, set _c_ to *true*; else, set _c_ to *false*.
1. If _condition_, set _d_ to *true*; else, set _d_ to *false*.

than to write

1. If _condition_, set _a_ to *true*.
1. Else, set _a_ to *false*.
1. If _condition_, set _b_ to *true*.
1. Else, set _b_ to *false*.
1. If _condition_, set _c_ to *true*.
1. Else, set _c_ to *false*.
1. If _condition_, set _d_ to *true*.
1. Else, set _d_ to *false*.

But if you disagree, or if my syntax is not normal for the spec, I will change it.

Reply:
I agree that it's more readable, but I am unsure if (1) this is a spec convention (2) it even matters.

Maybe we can ask the 262 editors about it?

Reply:
@syg @anba thoughts?

Reply:
Ping @anba @syg

Reply:
Both alternatives are valid and are used in ECMA-262 and ECMA-402. 

I think as a rule of thumb, I normally just try to follow the usage in the surrounding code resp. operations. Except when the condition is overly long or in a deeply nested step, so that the short form would either be rendered across multiple lines or when readability suffers.